### DISCUSSION TITLE
Add support to scan for Unleashed API tokens

### DISCUSSION THREAD

**Discussion by connorhsm** [community/community#69866] [Code Security] [answered]
### Select Topic Area

Question

### Body

I've found that Advanced Security Credential Scanning does not recognise API tokens committed to repository from the vendor Unleashed. https://support.unleashedsoftware.com/hc/en-us/articles/4403962084889-How-to-find-your-API-ID-and-Key-

**Comment by aashah** [comment, marked as answer]
Hey @connorhsm, I recommend reaching out to Unleashed and asking them to read over https://docs.github.com/en/code-security/secret-scanning/secret-scanning-partner-program#joining-the-secret-scanning-program-on-github
